azure-pipelines.yml

```diff
@@ -0,0 +1,73 @@
+trigger:
+- master
+
+jobs:
+  - job: Linux
+    pool:
+      vmImage: 'Ubuntu-16.04'
```

Comment:
just a question. Could it support multi-plat build. Example: 18.04 ?

Reply:
Only Ubuntu 16.04 is available, see https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#use-a-microsoft-hosted-agent

But other distributions could be supported via Docker.